src/services/AttachmentApi.js

```diff
@@ -38,10 +38,18 @@ export class AttachmentApi {
 	}
 
 	async createAttachment({ cardId, formData }) {
+		const config = {
+			onUploadProgress: function(progressEvent) {
+			  const percentCompleted = Math.round((progressEvent.loaded * 100) / progressEvent.total)
+			  console.log(percentCompleted)
+			},
+		}
+
 		const response = await axios({
 			method: 'POST',
 			url: this.url(`/cards/${cardId}/attachment`),
 			data: formData,
+			config: config,
```

Comment:
```suggestion
			...config,
```

As the first parameter to axios() is the config already.

Reply:
Thanks, now I get a console log result. 
How can I access this value from the view components?

Reply:
Well the easiest would be to pass a callback to the service method and then being able to access it directly in the component.